vagrant-provision.sh

```diff
@@ -0,0 +1,19 @@
+if [ ! -d /usr/local/Homebrew/.git ]; then
+  echo "==> Installing Homebrew ..."
+  /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  brew doctor
+else
+  echo "==> Homebrew is already installed"
+fi
+
+echo "==> Installing node and npm ..."
+brew install node
+
+echo "==> Installing Bundler ..."
+sudo gem install bundler -NV
```

Comment:
Is this sudo required?

Reply:
That depends on if you are using `rbenv`?

Reply:
Yes, it is required on a fresh install of masOS where no Ruby version manager is installed.

Reply:
Okay as long as the vagrant user has password-less sudo, and I think the first time you sudo you get a prompt saying you agree to perform the action.

Alternatively, you can `gem install --user-install bundler` and make sure `~/.gem/ruby/2.3.0/bin` is in the PATH

Reply:
Also, please use the long-form switches (e.g. `--no-document`) in scripts. It's much easier to read :)

Reply:
Done. PTAL.